engines/bastion_katello/app/assets/javascripts/bastion_katello/repositories/details/views/repository-info.html

```diff
@@ -141,7 +141,7 @@ <h4 translate>Basic Information</h4>
 
     </div>
 
-    <div class="detail">
+    <div class="detail" ng-hide="repository.content_type === 'puppet' || !repository.unprotected">
```

Comment:
I guess that is a good point.  We should probably add a comment "ubercert required" if its not unprotected